examples/server/public/index.html

```diff
@@ -670,6 +672,24 @@
         `
       };
 
+      const BoolField = ({ label, name, value }) => {
+        if (value) {
+          return html`
+            <div>
+              <label for="${name}">${label}</label>
+              <input type="checkbox" id="${name}" name="${name}" value="${value}" checked onclick=${updateParamsBool} />
```

Comment:
I'm new to preact, but I wonder if here we can dynamically populate `checked` prop instead of duplicating this HTML code? Something like this:

```suggestion
              <input type="checkbox" id="${name}" name="${name}" value="${value}" ${value ? 'checked' : ''} onclick=${updateParamsBool} />
```

Reply:
Thanks for your comment. I didn't know anything about preact, so I thought that the checked attribute had to be missing in order for the box to be unchecked like regular html. But according to [this](https://preactjs.com/guide/v10/forms/#checkboxes--radio-buttons), you can use a boolean to control the checked state. So, I updated the code to use it. I verified that it still updates penalize_nl as expected.

Reply:
I'm not 100% sure if `checked="${value}"` works or not, because it get translated to raw html `checked="true"` for example, which is not valid in html. The syntax in the document that you sent is jsx syntax (remember, `jsx !== html`). Can you please double check to see if your version work as expected?

Reply:
Yes, I verified by looking at the the body of the request.

Body with checkbox unchecked:
```
{"stream":true,"n_predict":400,"temperature":0.7,"stop":["</s>","Llama:","User:"],"repeat_last_n":256,"repeat_penalty":1.18,"penalize_nl":false,"top_k":40,"top_p":0.95,"min_p":0.05,"tfs_z":1,"typical_p":1,"presence_penalty":0,"frequency_penalty":0,"mirostat":0,"mirostat_tau":5,"mirostat_eta":0.1,"grammar":"","n_probs":0,"min_keep":0,"image_data":[],"cache_prompt":true,"api_key":"","slot_id":-1,"prompt":"This is a conversation between User and Llama, a friendly chatbot. Llama is helpful, kind, honest, good at writing, and never fails to answer any requests immediately and with precision.\n\nUser: Hello\nLlama:"}
```

Body with checkbox checked:
```
{"stream":true,"n_predict":400,"temperature":0.7,"stop":["</s>","Llama:","User:"],"repeat_last_n":256,"repeat_penalty":1.18,"penalize_nl":true,"top_k":40,"top_p":0.95,"min_p":0.05,"tfs_z":1,"typical_p":1,"presence_penalty":0,"frequency_penalty":0,"mirostat":0,"mirostat_tau":5,"mirostat_eta":0.1,"grammar":"","n_probs":0,"min_keep":0,"image_data":[],"cache_prompt":true,"api_key":"","slot_id":-1,"prompt":"This is a conversation between User and Llama, a friendly chatbot. Llama is helpful, kind, honest, good at writing, and never fails to answer any requests immediately and with precision.\n\nUser: Hello\nLlama:"}